huawei_solar_battery_optimization.yaml

```diff
@@ -160,6 +160,8 @@ input_text:
     name: HSBO Recommendation for 16:00
   hsbo_recommended_working_mode:
     name: HSBO Recommended Working Mode
+  hsbo_device_id:
+    name: HSBO Device ID for Huawei Solar device
```

Comment:
We would need two here for the scripts / huawei actions later in the file. 
  
  hsbo_device_id_inverter:
    name: HSBO Device ID for Huawei Solar device - Inverter
   hsbo_device_id_batteries:
    name: HSBO Device ID for Huawei Solar device - Batteries